terraform/provider.tf

```diff
@@ -0,0 +1,3 @@
+provider "google" {
```

Comment:
correct with TF 13+ style.
for example:
`terraform {  
  required_version = ">= 0.13"  
  required_providers {    
    google = {      
       source  = "hashicorp/google"      
       version = ">= 3.43, <4.0"    
}
}`